FF1Lib/ItemLists.cs

```diff
@@ -138,6 +138,13 @@ public static class ItemLists
 				.Where(x => x >= Item.Gold10 && x <= Item.Gold65000)
 				.ToList();
 
+		public static readonly IReadOnlyCollection<Item> BigGoldTreasure =
+			Enum.GetValues(typeof(Item))
+				.Cast<Item>()
+				.Where(x => x >= Item.Gold5000 && x <= Item.Gold65000)

```

Comment:
Careful, when price scaling is enabled the values can wildly change

Reply:
Question beeing, if that is a problem. I took 5000 because it was somewhat in the middle an a nice number. What would you propose?

Reply:
What I mean is the item Gold5000 can turn out to be 2500 GP or 10,000 GP with price scaling 50-200, and the item Gold3400 can end up more valuable than Gold5000, basically the name don't represent the real value, it's something to be aware of

Reply:
i see, to be corrected then, is there an easy way to check the actual values?

Reply:
The final value is the price, so you can probably cross-reference it (see ScalePrices in Scale.cs) :
		public const int PriceOffset = 0x37C00;
		public const int PriceSize = 2;
		public const int PriceCount = 240;

Reply:
ok

Reply:
should have been 10000g in the commit message